src/mbgl/tile/tile_data.hpp

```diff
@@ -82,6 +84,16 @@ class TileData : private util::noncopyable {
     virtual void redoPlacement(PlacementConfig, const std::function<void()>&) {}
     virtual void redoPlacement(const std::function<void()>&) {}
 
+    virtual void queryRenderedFeatures(
+            std::unordered_map<std::string, std::vector<std::string>>&,
```

Comment:
Unless their purpose is obvious from the type, provide names for parameter declarations. This applies to all the parameters here.